simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -66,20 +66,20 @@ public PushGateway(String address) {
   }
 
   /**
-   * Pushes all metrics in a registry, replacing all those with the same job as the grouping key.
+   * Pushes all metrics in a registry, replacing all those with the same job and no grouping key.
```

Comment:
That's just offering them as optional parameters. The grouping key comprises the job label plus whatever other optional labels you add.